config.lua

```diff
@@ -0,0 +1,302 @@
+Config = {}
+
+Config.useapartments = true -- false to disable apartments
+Config.useproperties = true -- false to disable properties
+
+-- Minimum price for a property (Avoid Players making their own properties for 1$ for example, i guess?)
+Config.MinimumPrice = 1000
+Config.GardenAndPoolTaxes = true
+Config.Taxes = { -- % of the property price (if price is 1000$ and tax is 10%, then the player pays 1100$), applies to both rent (repeatedly) and buy (once)
+    Garden = 3,
+    Pool = 5,
+    General = 10,
+}
+
+Config.Properties = {

```

Comment:
Is every property going to have a blip? Seems like we might need a config option to disable, or just not have this at all as this could get out of hand pretty quickly.

Reply:
only the properties you have access to not all of them 😄